string query
---
string[] names
OntologeniusSparqlResponse[] results
string error
